### DISCUSSION TITLE
How to manage API tokens?

### DISCUSSION THREAD

**Discussion by yoo** [semaphoreui/semaphore#2409] [Q&A] [answered]
Hello,
I'm looking into the Semaphore UI API and there is an endpoint to manage API tokens.
But I cannot find a way to manager API tokens through the web interface. Am I blind or is this a missing feature?

**Comment by jengstrom440** [comment]
The API reference docs here https://semaphoreui.com/api-docs/#/ have some api calls you can do to your server, but I don't think there is anything that you can do inside the web interface, but I may be mistaken. you can do something like http://yourserver.yourdomain:3000/api/user/tokens in a web browser to fetch your tokens on your account or with something like postman use a post to create a token. Hope that helps!

**Comment by fiftin** [comment, marked as answer]
https://docs.semaphoreui.com/administration-guide/api/
